fuel-core/src/chain_config.rs

```diff
@@ -49,10 +53,14 @@ impl ChainConfig {
             .map(|_| {
                 let secret = fuel_core_interfaces::common::fuel_crypto::SecretKey::random(&mut rng);
                 let address = Address::from(*secret.public_key().hash());
+                let bech32_data = Bytes32::new(*address).to_base32();
```

Comment:
Wait, this is making a bech32 out of the address, not the pubkey. If that's the case (is it?), then we don't actually need this change, as a downstream tool can do this conversion.

That being said, it may be a worthwhile change regardless since all downstream utilities use bech32.

Reply:
After investigating how SDK handles this, I came up with this implementation. And yes it is created out of `Address`. I double checked if the output addresses are correct by signing a transaction using initial accounts. Btw to do so, I needed to add an option to `forc-wallet` sign to accept a private key (since initial accounts are not generated from `forc-wallet` and we do not have the phrase for them, we cannot import the wallet. Instead I used the private key printed at the initial output to sign the test transaction). 

This conversion can be done in downstream utilities as you mentioned but outputting this as the default would be good for signing transactions with initial accounts while using local clients. If that is not important we can close this PR too